README.md

```diff
@@ -1,5 +1,5 @@
 
-# npm-diff(1)
```

Comment:
`1` doesn't refer to the version, but that it's a user level tool, which is a convention known from unix systems.